camelot/admin/action/field_action.py

```diff
@@ -296,3 +296,74 @@ def model_run(self, model_context):
             destination.write(storage.checkout_stream(stored_file).read())
 
 
+class AddNewObject( FieldAction ):
+    """Add a new object to a collection. Depending on the
+    'create_inline' field attribute, a new form is opened or not.
+
+    This action will also set the default values of the new object, add the
+    object to the session, and flush the object if it is valid.
+    """
+
+    shortcut = QtGui.QKeySequence.New
+    icon = Icon('plus-circle') # 'tango/16x16/actions/document-new.png'
+    tooltip = _('New')
+    verbose_name = _('New')
+    name = 'new_object'
+
+    def get_admin(self, model_context):
+        """
+        Return the admin used for creating and handling the new entity instance with.
+        By default, the given model_context's admin is used.
+        """
+        return model_context.admin
```

Comment:
in de field action gaat dit vermoedelijk ook de admin zijn van de parent object ipv die vh created object

Reply:
done (werd nog met ListActionModelContext aangeroepen).